tests/integ/test_dataframe.py

```diff
@@ -1321,6 +1321,10 @@ def test_join_left_outer(session):
     assert sorted(res, key=lambda r: r[0]) == expected
 
 
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="length of StringType is not supported in Local Testing",
+)
 def test_join_on_order(session):
```

Comment:
you don't need to skip it, you can choose the value depending on local_testing_mode
```def test_join_on_order(session, local_testing_mode):
    length = a if local_testing_mode else b
```